tests/Feature/PinThreadsTest.php

```diff
@@ -56,22 +56,16 @@ public function pinned_threads_are_listed_first()
         $threads = create(Thread::class, [], 3);
         $ids = $threads->pluck('id');
 
-        $this->getJson(route('threads'))->assertJson([
-            'data' => [
-                ['id' => $ids[0]],
-                ['id' => $ids[1]],
-                ['id' => $ids[2]]
-            ]
-        ]);
-        
+        $response_data = $this->getJson(route('threads'))->decodeResponseJson()['data'];
+        $this->assertEquals($ids[0], $response_data[0]['id']);
+        $this->assertEquals($ids[1], $response_data[1]['id']);
+        $this->assertEquals($ids[2], $response_data[2]['id']);
+
```

Comment:
Without this, travis is angry (https://travis-ci.org/JeffreyWay/council/jobs/340436609) it looks like it parses json differently :(